package.json

```diff
@@ -52,7 +52,7 @@
 		"@nextcloud/dialogs": "^3.1.4",
 		"@nextcloud/event-bus": "^3.0.0",
 		"@nextcloud/initial-state": "^2.0.0",
-		"@nextcloud/l10n": "^1.6.0",
+		"@nextcloud/l10n": "^2.0.1",
```

Comment:
I would stay with ^1.6.0 for now. The dependabot PR for the current nextcloud/vue version does not work https://github.com/nextcloud/nextcloud-vue/pull/3667.

Reply:
`Cannot find module '@nextcloud/l10n/dist/gettext.js' from 'src/l10n.js'`

This entry point was never documented to work, as the documentation said `@nextcloud/l10n/gettext`.
So 2.x dropped that entry and only allowed importing `@nextcloud/l10n` or `@nextcloud/l10n/gettext`.

This is changed [here](https://github.com/nextcloud/nextcloud-vue/blob/5663c718a7940057b6063024a64110b63484da10/src/l10n.js#L1) and works :)